specs/Picker-API_spec.md

```diff
@@ -0,0 +1,115 @@
+# Background
+
+The current FilePicker API for UWP has several limitations. Some of the top limitations 
+are as described below:
+
+**1. Unable to pick files and folders in a single instance of the Picker dialog**
+
+   The Picker dialog allows developers to specify picker mode to be single file or multiple
+   files. It does not however, allow developers to specify a mode to pick both files and folders.
+   There has been feedback indicating that this behaviour is limiting, requiring developers
+   to build their own picker experience. 
+
+**2. Unable to pick multiple folders.**
+
+   The Picker dialog (FolderPicker) does not allow users to pick multiple folders. The developer
+   experience currently is to select a single folder and repeat the dialog for more folders. This 
+   is a less than ideal experience for application developers.
+
+The Reunion picker dialog is going to address the above issues. This Reunion API will be based upon the below
+APIs in the Windows SDK. The Reunion Picker API will remove the deprecated APIs and add additional methods
```

Comment:
> will remove the deprecated APIs

Is it more correct to say that the Project Reunion APIs _supercede_ the Windows APIs? Deprecating APIs in Windows is a very long process.